src/PagePropsStatementCountLookup.php

```diff
@@ -14,13 +14,15 @@ public function __construct( LoadBalancer $loadBalancer ) {
 	}
 
 	public function getStatementCount( EntityId $entityId ) {
+		global $wgWBRepoSettings;
```

Comment:
Does this extension depend on Wikibase Repo? Not seeing that in composer.json. Then again, there is no wikibase repo specific package (which makes depending on it all the more insane)

If it does not, then you presumably want to default in this code